plugins/woocommerce/changelog/fix-php-8.1-null-args-deprecation

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Adjust wc-page-functions.php null args deprecations in PHP 8.1
```

Comment:
Will wait on an answer to [Jorge's question](https://github.com/woocommerce/woocommerce/pull/38635#issuecomment-1581012757) (might need a change here).